generators/ci-cd/command.ts

```diff
@@ -133,7 +133,7 @@ const command: JHipsterCommandDefinition = {
     },
     sonarName: {
       prompt: {
-        when: answers => includesValue('ciCd', ['jenkins'])(answers) && includesValue('ciCdIntegrations', ['ciCdIntegrations'])(answers),
+        when: answers => includesValue('ciCd', ['jenkins'])(answers) && includesValue('ciCdIntegrations', ['sonar'])(answers),
```

Comment:
@mshima It should includes also 'github', 'gitlab', 'travis'?

Reply:
I didn’t find any usage.
https://github.com/search?q=repo%3Ajhipster%2Fgenerator-jhipster%20sonarName&type=code